supervisor/addons/__init__.py

```diff
@@ -80,28 +80,47 @@ async def load(self) -> None:
 
     async def boot(self, stage: str) -> None:
         """Boot add-ons with mode auto."""
-        tasks = []
+        tasks: List[Addon] = []
         for addon in self.installed:
             if addon.boot != BOOT_AUTO or addon.startup != stage:
                 continue
-            tasks.append(addon.start())
+            tasks.append(addon)
 
+        # Evaluate add-ons which need to be started
         _LOGGER.info("Phase '%s' start %d add-ons", stage, len(tasks))
-        if tasks:
-            await asyncio.wait(tasks)
-            await asyncio.sleep(self.sys_config.wait_boot)
+        if not tasks:
+            return
```

Comment:
If there are no tasks, we are not calling `await asyncio.sleep(self.sys_config.wait_boot)`. Is that on purpose?

Reply:
It's like before if there is nothing to do, we need also not spend time waiting until services are up